lesson07/jsonRPC/client/jsonClient.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"net/rpc/jsonrpc"
+	"os"
+
+	"github.com/SergeiLisitsyn/gocourse/lesson07/jsonRPC/common"
+)
+
+var reply common.Item
+var list common.List
+
+func main() {
```

Comment:
This works, but I would prefer something like an interactive client:
A user would get a prompt to choose between two actions: 
1. List contacts
2. Add a new contact

The first one would return the results, preferably formatted, rather then just print out the map
The second would ask for another prompt to request a name to register
After an action is done, you can choose another action.
So, you would have a for loop with a switch inside, that performs one action or another in each switch case or prints out a warning if incorrect value was entered in your default case (for example action 3, which doesn't exist)

Reply:
json Client is refactored. In rest task i have made search.go file with new handler and added the handler in main.go file.